pkg/controllers/disruption/emptynodeconsolidation.go

```diff
@@ -55,15 +56,17 @@ func (c *EmptyNodeConsolidation) ComputeCommand(ctx context.Context, disruptionB
 		if len(candidate.reschedulablePods) > 0 {
 			continue
 		}
-		if disruptionBudgetMapping[candidate.nodePool.Name] == 0 {
+		if disruptionBudgetMapping.IsZero(candidate.nodePool.Name, v1beta1.DisruptionReasonUnderutilized) {
```

Comment:
Left this as underutilized because its associated with the consolidation mode WhenUnderutilized but it might make sense to have it be v1beta1.DisruptionReasonEmpty